integrations/access/jira/client.go

```diff
@@ -163,6 +163,18 @@ func statusFromStatusCode(httpCode int) types.PluginStatus {
 	return &types.PluginStatusV1{Code: code}
 }
 
+// buildSummary creates the Issue's summary by using the user name and roles.
+// No official docs seem to exist, but it's _known_ that summary field must be less than 255 chars:
+// Eg https://community.atlassian.com/t5/Jira-questions/Summary-must-be-less-than-255-characters/qaq-p/989632
+func buildSummary(reqData RequestData) string {
+	summary := fmt.Sprintf("%s requested %s", reqData.User, strings.Join(reqData.Roles, ", "))
+	if len(summary) <= 254 {
+		return summary
+	}
+
+	return summary[:strings.LastIndexAny(summary[:254], ",")]
```

Comment:
Should we just say "my user request 12 roles" instead of truncating the role list?